include/coro_rpc/coro_rpc/coro_connection.hpp

```diff
@@ -458,9 +487,11 @@ class coro_connection : public std::enable_shared_from_this<coro_connection> {
   size_t body_size_ = 256;
   std::vector<char> body_;
   // FIXME: queue's performance can be imporved.
-  std::deque<std::vector<char>> write_queue_;
+  std::deque<std::pair<std::string, std::string>> write_queue_;
   std::errc rsp_err_;
   bool delay_ = false;
+  bool is_read_coro_done = true, is_write_coro_done = true;
```

Comment:
1. is_read_coro_done --> is_read_coro_done_, is_write_coro_done -->is_write_coro_done_
2. don't define them in one line code.